Dev 1.1.1 (#14)
3MFConsortium/spec_booleans: 3MF Boolean operations.md

**operation** - The boolean operation to perform. The options for the boolean shapes are the following:

> 1.	*union*. The resulting object shape is defined as the merger of the shapes. The resulting object surface property is defined by the property of the surface property defining the outer surface, as defined by [the 3MF Core Specification overlapping order](https://github.com/3MFConsortium/spec_core/blob/1.4.0/3MF%20Core%20Specification.md#412-overlapping-order). The material and the volumetric properties in the overlapped volume are defined by the ones in last overlapping object in that volume, or none if last overlapping object has no material or volumetric property defined. 

Author: ```suggestion
1.	*union*. The resulting object shape is defined as the merger of the shapes. The resulting object surface property is defined by the property of the surface property defining the outer surface, as defined by [the 3MF Core Specification overlapping order](https://github.com/3MFConsortium/spec_core/blob/1.4.0/3MF%20Core%20Specification.md#412-overlapping-order). The material and volumetric properties in the overlapping volume are determined by the properties of the last overlapping object in that volume. If the last overlapping object does not have material or volumetric properties defined, then no properties are assigned to the overlapping volume.
```

    union(base,a,b,c) = base Ս (a Ս b Ս c) = ((base Ս a) Ս b) Ս c

2.  *difference*. The resulting object shape is defined by the shape in the base object shape that is not in any other object shape. The resulting object surface property, where overlaps, is defined by the object surface property of the subtracting object(s), or no-property when the subtracting object has no property defined in that surface.

    difference(base,a,b,c) = base - (a Ս b Ս c) = ((base - a) - b) - c

> 3.  *intersection*. The resulting object shape is defined as the common (clipping) shape in all objects. The resulting object surface property is defined as the object surface property of the object defining the new surface, or no-property when that object has no property defined in the new surface. The material and the volumetric properties in the overlapped volume are defined by the ones in the in last overlapping object in that volume, or none if last overlapping object has no material or volumetric property defined. 

Reviewer: ```suggestion
3.  *intersection*. The resulting object shape is defined as the common (clipping) shape in all objects. The resulting object surface property is defined as the object surface property of the object defining the new surface, or no-property when that object has no property defined in the new surface. The material and volumetric properties in the overlapping volume are determined by the properties of the last overlapping object in that volume. If the last overlapping object does not have material or volumetric properties defined, then no properties are assigned to the overlapping volume.
```

Author: Thanks @3dJan  for the suggestion. Since the spec was already approved as it is, I'll hold the final publishing until we review your proposal next call.
